TestingTool/task.py

```diff
@@ -16,8 +16,12 @@ class Task(EmbeddedDocument):
     def run(self, browser_instance):
         data = dict()
         result = False
-        result, data = ACTION_LIST[self.action](self.args, browser_instance)
-        if self.reverse_result:
-            self.result = Result(not result, data)
-        else:
-            self.result = Result(result=result, data=data)
+        try:
+            result, data = ACTION_LIST[self.action](self.args, browser_instance)
+        except Exception as error:
+            self.result = Result(False, data=str(error))
+        finally:
```

Comment:
Nu este necesar un block finally aici, intrucat finally se executa indiferent de ce se intampla in try/except, si atunci rezultatul in cazul in care da exceptie va fi suprascris.
Ce ar trebui facut e ca doar in cazul in care a rulat cu succes sa facem if/else, si asta putem obtine cu 'else', care se executa doar daca 'nu a dat exceptie'